tests/testthat/test-survival-tune-bayes.R

```diff
@@ -39,7 +35,12 @@ test_that("Bayesian tuning survival models with static metric", {
 
   gctrl <- control_grid(save_pred = TRUE)
   bctrl <- control_bayes(save_pred = TRUE)
+
   # standard setup end
+  # ------------------------------------------------------------------------------
+  # Bayes with static metrics
```

Comment:
```suggestion
  # Bayes with static metrics --------------------------------------------------
```

If you write the comment line like this, then RStudio will pick up on it in the outline, which is quite nice for these longer repeated test files

<img width="579" alt="Screenshot 2023-11-15 at 11 17 23 AM" src="https://github.com/tidymodels/extratests/assets/14034784/933e7ea8-2a22-4fc4-bc94-45833be8e2d7">

Reply:
that's horrid but ok 😄

Reply:
```r
make_banner <- function(text, width = 78) {
  y <- paste("#", text)
  n <- nchar(y)
  y <- paste(y, paste0(rep("-", width - n - 1), collapse = ""))
  cat(y)
  invisible(y)
}

make_banner("test metric collection")
```